common/src/IO/AssimpParser.cpp

```diff
@@ -350,6 +350,58 @@ void processNode(
   }
 }
 
+#define AI_MDL_HL1_NODE_BODYPARTS "<MDL_bodyparts>"
```

Comment:
I'll go with your suggestion to keep consistent code style, but interestingly my IDE tells me that the `static` keyword is not required in this case since it's in an anonymous namespace:

![image](https://github.com/TrenchBroom/TrenchBroom/assets/3071180/77687b04-7d8e-455d-b471-96f5a3d054c2)

Reply:
I didn't notice that. If it's in an anonymous namespace you can ditch the static keyword, yes.